firestore/test/firestoreTest.php

```diff
@@ -74,13 +74,15 @@ public static function tearDownAfterClass(): void
 
     public function testInitialize()
     {
-        $output = $this->runFirestoreSnippet('setup_client_create', []);
+        $output = self::runFunctionSnippet('setup_client_create', [self::$projectId]);
         $this->assertStringContainsString('Created Cloud Firestore client with default project ID.', $output);
```

Comment:
This test wasn't updated
```suggestion
        $this->assertStringContainsString('Created Cloud Firestore client with project ID: ', $output);
```